catalog/transformer/catalog-transformer-bootflag/src/main/java/org/codice/ddf/catalog/transformer/bootflag/InputTransformerBootServiceFlag.java

```diff
@@ -81,7 +77,7 @@ public InputTransformerBootServiceFlag(
     this.inputTransformers = inputTransformers;
     this.executorService = Executors.newSingleThreadExecutor();
     this.transformerCheckPeriodMillis = DEFAULT_TRANSFORMER_CHECK_PERIOD_MILLIS;
-    this.transformerWaitTimeoutMillis = getTransformerWaitTimeout();
+    this.transformerWaitTimeoutMillis = DEFAULT_TRANSFORMER_WAIT_TIMEOUT_MILLIS;
```

Comment:
How about `private static final long TRANSFORMER_CHECK_PERIOD_MILLIS = TimeUnit.MINUTES.toMillis(20);` instead of the line in the constructor and the multiple class variables?